docs/content/tutorials/tutorial-a-first-look-at-druid.md

```diff
@@ -76,7 +76,7 @@ You should see a bunch of files:
 
 ## External Dependencies
 
-Druid requires 3 external dependencies. A "deep storage" that acts as a backup data repository, a "metadata storage" such as MySQL to hold configuration and metadata information, and [Apache Zookeeper](http://zookeeper.apache.org/) for coordination among different pieces of the cluster. For this tutorial, we only need the zookeeper dependency.
+Druid requires 3 external dependencies. A "deep storage" that acts as a backup data repository, a "metadata storage" such as MySQL to hold configuration and metadata information. Druid uses [Apache Derby](http://db.apache.org/derby/) by default. And, [Apache Zookeeper](http://zookeeper.apache.org/) for coordination among different pieces of the cluster. For this tutorial, we only need the zookeeper dependency.
```

Comment:
I suggest that the flow (and grammar) would be improved here by using a bullet list for the three dependencies.  
That also applies to the same change in tutorial-the-druid-cluster.md.